src/NumericalAlgorithms/DiscontinuousGalerkin/SimpleMortarData.hpp

```diff
@@ -43,6 +43,15 @@ class SimpleMortarData {
     return *local_data_;
   };
 
+  /// Retrieve the remote data at `temporal_id`
+  const RemoteVars& remote_data(const TemporalId& temporal_id) const noexcept {
+    ASSERT(local_data_, "Remote data not available.");
```

Comment:
`remote_data_`

Reply:
fixed (squashed the small change directly)